app/wss.js

```diff
@@ -64,6 +64,16 @@ const onReceiveDataFromClient = (clientId, byteData) => {
             currentRoom = game.getCurrentRoom();
             sendToAllClients(HumanRoomUpdateData(currentRoom));
             break;
+        case HUMAN_ACTIONS.switchHumanTool:
+            const newTool = action.args.tool;
```

Comment:
We've been fast and loose with input validation so far. It's time we clean that up. 
I made an issue for it: #102